apps/user/src/services/form/출신학교및학력/api.ts

```diff
@@ -0,0 +1,9 @@
+import axios from 'axios';
+
+export const schoolList = async (name: string) => {
+    const { data } = await axios.get(
+        `${process.env.NEXT_PUBLIC_SCHOOL_LIST_API!}&SCHUL_NM=${name}`,
```

Comment:
```suggestion
    const { data } = await maru.get(
        `&SCHUL_NM=${name}`,
```

maru customAxios 사용하면 줄일수있어요~

Reply:
이렇게 한 까닭은 해당 axios는 마루 백엔드의 api가 아닌 나이스의 오픈 API를 사용했기 때문입니다.

Reply:
근데 굳이 가린이유는 무엇인가요? 나이스 api는 open api기 때문에 굳이 가릴필요가없어보입니다 명재님의 생각이 궁금합니다 !